monai/metrics/utils.py

```
@@ -293,6 +326,8 @@ def get_edge_surface_distance(
     use_subvoxels: bool = False,
     symmetric: bool = False,
     class_index: int = -1,
+    mask: torch.Tensor | None = None,
+    ignore_index: int | None = None,
```

Ruff (0.15.5)

[warning] Unused function argument: `ignore_index`

(ARG001)